draft-ietf-rats-endorsements.md

```diff
@@ -201,16 +201,40 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
-
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+# Key Material for Attestation
+
+A Verifier receives Evidence from an Attester that must be considered untrusted
+until verified through cryptography. Typically,
```

Comment:
```suggestion
A Verifier receives Evidence from an Attester that must be considered untrusted
until its roots of trust endorsements and their corresponding identity documents are verified through cryptography. Typically,
```

Reply:
Sorry, I read from the top down to this Section and am unsure which other text you refer to. Could you point me to the right place, please?